src/api/services/format-date-service.ts

```diff
@@ -0,0 +1,21 @@
+import {Injectable} from '@angular/core';
+import moment from 'moment';
+
+@Injectable()
+export class FormatDate {
+
+    constructor() {}
+
+    setLang(lang:string, messages:Object):void {
+        moment.locale(lang, messages || {});
+    }
+
+    getLang() {
+        return moment.locale();
+    }
+
+    getRelative(time):string {
```

Comment:
Not sure if instead of a service we can use a pipe for formatting the date. Ideas?